### PR TITLE
feat: update sidebar's menuicon

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.
update meunicon
image

**Related issues**

fix/resolve #0001

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]

File: web/src/helpers.tsx [hunk 1]
```diff
@@ -45,17 +46,17 @@ export const getMenuData = (): MenuDataItem[] => {
...
       name: 'service',
       path: '/service/list',
-      icon: <IconFont name="iconconsumer" />,
+      icon: <IconFont name="iconservice" />,
     },
     {
       name: 'consumer',
       path: '/consumer/list',
-      icon: <IconFont name="iconconsumer" />,
+      icon: <IconFont name="iconconsumer1" />,
```
Comment (liuxiran) [inline web/src/helpers.tsx:54]: it seems that `iconconsumer` is used, how about remove the old `iconconsumer`, and rename the new one `iconconsumer1` to `iconconsumer`, that would be more clear

**Comment by nic-chen** [comment]
ping

**Comment by juzhiyuan** [comment]
just regenerated the icons, waiting for upload.

File: web/src/helpers.tsx (hunk 1 above)
Comment (LiteSun) [reply web/src/helpers.tsx]: Sure.

**Review by liuxiran — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Review by Jaycean — APPROVED** [review]

**Pull request merged**